contrib/widget-examples/lemonbar/lang.lua

```diff
@@ -0,0 +1,16 @@
+dofile 'lib.lua'
+
+widget = {
+  plugin = 'xkb',
+  cb = function(t)
+    local text = ''
```

Comment:
No sense in setting it to an empty string, you can just define it as `local text`. Also, it can be rewritten like this:
```lua
local text = ({['us'] = ' En ', ['ru'] = ' Ru '})[t.name] or ' ?? '
```